### ISSUE TITLE
Result Miou 

### ISSUE THREAD

**Issue by cuishuhao** [jfzhang95/pytorch-deeplab-xception#15]
could anyone show the training and testing result ?
my result goes too bad to see.

**Comment by edgarmedina1801** [comment]
I tried some epochs but I didn't get good results. 
My metrics are:
Loss: 281652.201770, miou: 0.3599

**Comment by jfzhang95** [comment]
I think you can train model with more epoch because
modified xception do not have a pretrained model and 
also you can use tensorboardX to visualize outputs of 
training data.

**Comment by edgarmedina1801** [comment]
well, I am training the model using the original configuration (including the pre-trained model you used) and it is in the epoch 50. I attach the current results.

```
[Epoch: 49, numImages: 10582]
Loss: 254079.521480
Execution time: 1499.6592651726678
Save model at .../deeplabv3plus-xception-voc_epoch-49.pth
Validation:
[Epoch: 49, numImages:  1449]
Loss: 243406.573831
MIoU: 0.387194

```

**Comment by jfzhang95** [comment]
Did you use the latest code to train the model?
Network architecture defined in previous version has some problems.

**Comment by jfzhang95** [comment]
Thank you for your patience and interest in this code.

Further discussion about performance of this code can be moved to #12 .
